cpp/tests/hashing/hash_test.cpp

```diff
@@ -129,6 +129,229 @@ TEST_F(HashTest, MultiValueNulls)
   CUDF_TEST_EXPECT_COLUMNS_EQUAL(spark_output1->view(), spark_output2->view());
 }
 
+TEST_F(HashTest, BasicList)
+{
+  using lcw = cudf::test::lists_column_wrapper<uint64_t>;
```

Comment:
This pattern is written with `LCW` in caps everywhere else in the code base. Same for `ICW`.